lib/secrets.rb

```diff
@@ -158,4 +161,8 @@ def read_vault_token(authfile_path)
   rescue OpenSSL::X509::CertificateError
     File.read(authfile_path)
   end
+
+  def log(msg)
+    puts "#{Time.now.to_i}: #{msg}" unless ENV["testing"]
```

Comment:
can be whatever format people want.   not sure if the sidecar stuff get's logged anywhere else.